sherpa/astro/flux.py

```diff
@@ -345,14 +367,19 @@ def decompose(mdl):
 
 def sample_flux(fit, data, src,
                 method=calc_energy_flux, correlated=False,
-                num=1, lo=None, hi=None, numcores=None, samples=None):
+                num=1, lo=None, hi=None, numcores=None, samples=None,
+                clip="hard"):
```

Comment:
For consistency, shouyld "hard" be 'hard'.  Looks like single quotes are used elsewhere...